Sources/BrowserServicesKit/Autofill/AutofillUserScript+Email.swift

```diff
@@ -65,8 +66,30 @@ extension AutofillUserScript {
     func emailGetAlias(_ message: UserScriptMessage, _ replyHandler: @escaping MessageReplyHandler) {
         guard let dict = message.messageBody as? [String: Any],
               let requiresUserPermission = dict["requiresUserPermission"] as? Bool,
-              let shouldConsumeAliasIfProvided = dict["shouldConsumeAliasIfProvided"] as? Bool else { return }
+              let shouldConsumeAliasIfProvided = dict["shouldConsumeAliasIfProvided"] as? Bool,
+              let isIncontextSignupAvailable = dict["isIncontextSignupAvailable"] as? Bool else { return }
 
+        let signedIn = emailDelegate?.autofillUserScriptDidRequestSignedInStatus(self) ?? false
```

Comment:
Maybe we can use a guard/switch here to remove the neted ifs..  also worth writing a test for this method?